docs/_include/card/timeseries-dask.md

```diff
@@ -0,0 +1,23 @@
+::::{info-card}
+
+:::{grid-item}
+:columns: auto 9 9 9
+**Notebook: How to Build Time Series Applications with CrateDB**
+
+This notebook illustrates how to import and work with time series data in CrateDB.
+It uses Dask to import data into CrateDB.
```

Comment:
Link to https://docs.dask.org/en/stable/dataframe.html?

Reply:
Thanks. Please note we are maintaining a _gallery of info cards about integrations_ within [crate-clients-tools](https://github.com/crate/crate-clients-tools) aka. the [Ecosystem Catalog](https://cratedb.com/docs/crate/clients-tools/) [^1]. We can and should talk about how that can converge better on behalf of a future refactoring operation, probably into having a single place of truth [^2].

[^1]: https://cratedb.com/docs/crate/clients-tools/en/latest/connect/df.html#dask
[^2]: https://github.com/crate/tech-content/issues/117